frontend/package.json

```diff
@@ -13,7 +13,8 @@
     "react-redux": "^7.1.1",
     "react-scripts": "3.2.0",
     "redux-logger": "^3.0.6",
-    "redux-saga": "^1.1.1"
+    "redux-saga": "^1.1.1",
+    "uuid": "latest"
```

Comment:
can we get rid of this please

Reply:
What exctly? I'm using uuid for the ids.